Presentation.Web/Tests/01_Parallel/data-processing/tabs/data-processing-registration.it-systems.e2e.spec.ts

```diff
@@ -39,8 +37,9 @@ describe("Data processing registration it-systems test", () => {
     it("Assigning and removing systems",
         () => {
             const dprName = createName(3);
-
-            dpaHelper.createAndOpenDataProcessingRegistration(dprName)
+            SystemCatalogHelper.createSystemAndActivateLocally(system1)
+                .then(() => SystemCatalogHelper.createSystemAndActivateLocally(system2))
+                .then(() => dpaHelper.createAndOpenDataProcessingRegistration(dprName))
```

Comment:
måske man lige skulle huske at logge ud og ind som lokal admin inden selve scenariet udføres... vi skal undgå at gennemføre lokale tests som global admin for de har det med at skjule fejl (global admin er jo gud)